### PR TITLE
csnd:SND: Reformat source code

### PR DESCRIPTION
**Pull request by mailwl**
Reformat the source similar other services.

### CODE REVIEW

**Comment by yuriks** [comment]
Can you split the formatting and the stubbing in separate commits?

File: src/core/hle/service/csnd_snd.cpp [hunk 1]
```diff
@@ -14,75 +14,163 @@
...
+static void FlushDataCache(Service::Interface* self) {
+    u32* cmd_buff = Kernel::GetCommandBuffer();
+    u32 address = cmd_buff[1];
+    u32 size = cmd_buff[2];
+    u32 process = cmd_buff[4];
+
+    cmd_buff[0] = IPC::MakeHeader(0x13, 1, 0);
+    cmd_buff[1] = RESULT_SUCCESS.raw; // No error
+
+    LOG_WARNING(Service_DSP, "called address=0x%08X, size=0x%X, process=0x%08X", address, size,
```
Comment (yuriks) [inline src/core/hle/service/csnd_snd.cpp]: Also change this to LOG_DEBUG

Comment (yuriks) [inline src/core/hle/service/csnd_snd.cpp]: Oh yeah, this is also using the wrong log category

Comment (mailwl) [inline src/core/hle/service/csnd_snd.cpp]: ehh, copy/paste is evil...

**Comment by mailwl** [comment]
should i add additional clean for shared_memory and mutex in ~Interface(){}?

File: src/core/hle/service/csnd_snd.cpp [hunk 2]
```diff
@@ -14,75 +14,139 @@
...
 
     u32 size = Common::AlignUp(cmd_buff[1], Memory::PAGE_SIZE);
+
     using Kernel::MemoryPermission;
     shared_memory = Kernel::SharedMemory::Create(nullptr, size, MemoryPermission::ReadWrite,
                                                  MemoryPermission::ReadWrite, 0,
                                                  Kernel::MemoryRegion::BASE, "CSND:SharedMemory");
 
     mutex = Kernel::Mutex::Create(false);
+    mutex->name = "CSND:mutex";
```
Comment (Subv) [inline src/core/hle/service/csnd_snd.cpp]: You can pass the name as a parameter to Create

**Comment by mailwl** [comment]
Done.

File: src/core/hle/service/csnd_snd.cpp [hunk 3]
```diff
@@ -96,7 +95,7 @@ static void ExecuteCommands(Service::Interface* self) {
 
     if (ptr == nullptr) {
         cmd_buff[1] = 1;
-        LOG_ERROR(Service_CSND, "called, can't get pointer to command, address=0x%08X", addr);
+        LOG_ERROR(Service_CSND, "called, can't get pointer to command, addr=0x%08X", addr);
```
Comment (lioncash) [inline src/core/hle/service/csnd_snd.cpp]: for what purpose?

File: src/core/hle/service/csnd_snd.cpp [hunk 4]
```diff
@@ -12,72 +12,138 @@
...
+ *  Inputs:
+ *      0 : Header Code[0x00010140]
+ *      1 : Shared memory block size, for mem-block creation
+ *  Outputs:
+ *      1 : Result of function, 0 on success, otherwise error code
+ *      2 : Handle-list header
+ *      3 : Mutex handle
+ *      4 : Shared memory block handle
+ */
+static void Initialize(Service::Interface* self) {
```
Comment (lioncash) [inline src/core/hle/service/csnd_snd.cpp]: `Service::` isn't necessary.

File: src/core/hle/service/csnd_snd.cpp [hunk 5]
```diff
@@ -12,72 +12,138 @@
...
-    u32* const cmd_buff = Kernel::GetCommandBuffer();
-    u8* const ptr = shared_memory->GetPointer(cmd_buff[1]);
+/**
+ * CSND_SND::Shutdown service function
+ *  Inputs:
+ *      0 : Header Code[0x00020000]
+ *  Outputs:
+ *      1 : Result of function, 0 on success, otherwise error code
+ */
+static void Shutdown(Service::Interface* self) {
```
Comment (lioncash) [inline src/core/hle/service/csnd_snd.cpp]: Ditto

File: src/core/hle/service/csnd_snd.cpp [hunk 6]
```diff
@@ -12,72 +12,138 @@
...
+/**
+ * CSND_SND::ExecuteCommands service function
+ *  Inputs:
+ *      0 : Header Code[0x00030040]
+ *      1 : Command offset in shared memory.
+ *  Outputs:
+ *      1 : Result of function, 0 on success, otherwise error code
+ *      2 : Available channel bit mask
+ */
+static void ExecuteCommands(Service::Interface* self) {
```
Comment (lioncash) [inline src/core/hle/service/csnd_snd.cpp]: Ditto

File: src/core/hle/service/csnd_snd.cpp [hunk 7]
```diff
@@ -12,72 +12,138 @@
...
-void AcquireSoundChannels(Interface* self) {
+/**
+ * CSND_SND::AcquireSoundChannels service function
+ *  Inputs:
+ *      0 : Header Code[0x00050000]
+ *  Outputs:
+ *      1 : Result of function, 0 on success, otherwise error code
+ *      2 : Available channel bit mask
+ */
+static void AcquireSoundChannels(Service::Interface* self) {
```
Comment (lioncash) [inline src/core/hle/service/csnd_snd.cpp]: Ditto

File: src/core/hle/service/csnd_snd.cpp [hunk 8]
```diff
@@ -12,72 +12,138 @@
...
+    {0x00060000, nullptr, "ReleaseSoundChannels"},
+    {0x00070000, nullptr, "AcquireCaptureDevice"},
+    {0x00080040, nullptr, "ReleaseCaptureDevice"},
+    {0x00090082, nullptr, "FlushDataCache"},
+    {0x000A0082, nullptr, "StoreDataCache"},
+    {0x000B0082, nullptr, "InvalidateDataCache"},
+    {0x000C0000, nullptr, "Reset"},
+};
+
+////////////////////////////////////////////////////////////////////////////////////////////////////
```
Comment (lioncash) [inline src/core/hle/service/csnd_snd.cpp]: This comment isn't necessary.

File: src/core/hle/service/csnd_snd.cpp (hunk 3 above)
Comment (mailwl) [inline src/core/hle/service/csnd_snd.cpp]: in original check to nullptr after usage shared_memory.
but now, i see, if shared_memory is not nullptr, then ptr can't be null too. i remove unnesessary check

**Comment by mailwl** [comment]
done

File: src/core/hle/service/csnd_snd.cpp (hunk 3 above)
Comment (lioncash) [inline src/core/hle/service/csnd_snd.cpp]: I was asking why you changed `address` to `addr` when it's quite obvious what it's referring to, not why the check was being performed.

Comment (mailwl) [inline src/core/hle/service/csnd_snd.cpp]: ah, text log and variable name was not the same

File: src/core/hle/service/csnd_snd.h [hunk 9]
```diff
@@ -18,18 +18,6 @@ class CSND_SND final : public Interface {
...
-};
-
-void Initialize(Interface* self);
-void ExecuteType0Commands(Interface* self);
-void AcquireSoundChannels(Interface* self);
-void Shutdown(Interface* self);
-
 } // namespace CSND
 } // namespace Service
+
```
Comment (bunnei) [inline src/core/hle/service/csnd_snd.h]: This extra line is breaking CI

**Pull request merged**